simuleval/evaluator/evaluator.py

```diff
@@ -245,11 +244,12 @@ def __call__(self, system):
         with open(
             self.output / "instances.log", "a"
         ) if self.output else contextlib.nullcontext() as file:
-            idx = 0
```

Comment:
The new cur_index variable, where is it initialized to 0?

Reply:
We are not initializing `cur_index` to 0 but to `start_index - 1` - https://github.com/fairinternal/seamless_communication/pull/118/files#diff-a404c48b410fd80711d9ff8bf3f705a9ea2ac9d53f533492104245e715e3dc20R51